composer.json

```diff
@@ -29,7 +29,7 @@
         "symfony-cmf/testing": "^3@dev"
     },
     "suggest": {
-        "symfony/event-dispatcher": "DynamicRouter can optionally trigger an event at the start of matching. Minimal version (^3.4 || ^4.3)"
+        "symfony/event-dispatcher": "DynamicRouter can optionally trigger an event at the start of matching. Minimal version (^3.4 || ^4.3 || ^5.0‡ )"
```

Comment:
good find! there is some weird character after the 5.0, can you remove that character?

Reply:
Good catch! Fixed!